assets/javascripts/discourse/helpers/format.js

```diff
@@ -0,0 +1,7 @@
+import Helper from "@ember/component/helper";
+
+export function format(number) {
+  return number.toLocaleString(undefined, {});
```

Comment:
`undefined` means we won't be passing the current interface language here. 

Does using `I18n.toNumber(number)` work (please also test with a language that uses a different separator, like German)?

Reply:
I thought undefined meant it would take the system settings. How would I18n.toNumber(number) work as a helper?

Reply:
This should do it. 

```suggestion
  return I18n.toNumber(number, { precision: 0 });
```